### ISSUE TITLE
Refactor the MNIST classifier example using the v1 Python client

### ISSUE THREAD

**Issue by keithmanville** [usnistgov/dioptra#596]
Refactor the MNIST classifier example with the v1 client

This includes pulling together the required plugins, refactoring the entrypoint yaml, and updating the notebook to use the new client.

Blocked by:
- #530

### Definition of done
- [ ] The MNIST classifier example is refactored and the notebook is updated
- [ ] The example jobs run successfully
- [ ] The example is merged into dev

**Comment by jkglasbrenner** [comment]
Currently blocked, waiting on NN to be merged.
